src/packageSourceGenerator/PackageSourceGenerator.proj

```diff
@@ -57,6 +44,23 @@
            Condition="('$(PackageName)' == '' or '$(PackageVersion)' == '') and '$(PackageCSV)' == ''" />
   </Target>
 
+  <!-- Parse the passed in CSV file into PackageInfo items with metadata. -->
+  <UsingTask TaskName="ParsePackagesCsv" AssemblyFile="$(PackageSourceGeneratorTaskAssembly)" />
```

Comment:
@ViktorHofer, I went with an approach to encapsulate the CSV logic within a task.  The problem is the PackageSourceGeneratorTask dependency project doesn't get in time the packageDownloads need to be defined.  Do you have any suggestions on how to get the PackageSourceGeneratorTask to build sooner?  TIA